Documentation/DSCResources/VMHostVss/VMHostVss.md

```diff
@@ -0,0 +1,70 @@
+# VMHostTpsSettings
+
+## Parameters
```

Comment:
The whole documentation should be updated with the properties of VMHostVSS Resource.